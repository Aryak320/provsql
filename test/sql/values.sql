\set ECHO none
\pset format unaligned
SET search_path TO provsql_test, provsql;

CREATE TABLE tvalues AS
SELECT
    *,
    formula (provenance (), 'personnel_name')
FROM (
    VALUES (1)) t,
    personnel;

SELECT
    remove_provenance ('tvalues');

SELECT
    *
FROM
    tvalues
ORDER BY
    id;
