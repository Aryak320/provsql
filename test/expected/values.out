\set ECHO none
remove_provenance

(1 row)
column1|id|name|position|city|classification|formula
1|1|John|Director|New York|unclassified|John
1|2|Paul|Janitor|New York|restricted|Paul
1|3|Dave|Analyst|Paris|confidential|Dave
1|4|Ellen|Field agent|Berlin|secret|Ellen
1|5|Magdalen|Double agent|Paris|top_secret|Magdalen
1|6|Nancy|HR|Paris|restricted|Nancy
1|7|Susan|Analyst|Berlin|secret|Susan
(7 rows)
